src/Popover.vue

```diff
@@ -1,14 +1,17 @@
 <template>
   <div>
```

Comment:
Let's change this to `<span>` so that the button will stay in the same line, which is similar to how the old version behaves.